src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -285,4 +286,34 @@ public static Dimension getBrowserSize() {
   public static String getDisableCommunityPageSalesPitchDialog() {
     return getProp("disableCommunityPageSalesPitchDialog");
   }
+
+  public static HttpHost getBorderProxy() {
+
+    EnvType envType = getEnvType(getEnv());
+    File configFile = new File(getCredentialsFilePath());
+
+    if (envType.equals(EnvType.DEV)) {
+      return new HttpHost(
+          XMLReader.getValue(configFile, "border.poz.address"),
+          Integer.parseInt(XMLReader.getValue(configFile, "border.poz.port")),
+          XMLReader.getValue(configFile, "border.poz.protocol")
+      );
+    } else {
+      return new HttpHost(
+          XMLReader.getValue(configFile, "border.sjc.address"),
+          Integer.parseInt(XMLReader.getValue(configFile, "border.sjc.port")),
+          XMLReader.getValue(configFile, "border.sjc.protocol")
+      );
+    }
+  }
+
+  //TODO: Get rid of replacing wikia with fandom
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)